test/messages/user-settings/index.test.js

```diff
@@ -58,4 +58,15 @@ describe('CombinedSettings rendering', () => {
     }).getAttachments();
     expect({ attachments }).toMatchSnapshot();
   });
+
+  test('doesn\'t work if temporarily muted prompts is after current time', async () => {
```

Comment:
Could we update this test name to be something more like "doesn't show if temporary mute is no longer active"